package.json

```diff
@@ -134,6 +134,16 @@
         ],
         "url": "./schemas/import_map.schema.json"
       }
+    ],
+    "snippets": [
+      {
+        "language": "javascript",
+        "path": "snippets/import_enhancement.json"
+      },
+      {
+        "language": "typescript",
+        "path": "snippets/import_enhancement.json"
+      }
```

Comment:
It would be great if we could dynamically load these depending on if `"deno.enable"` is `true`. If that is not possible, this is fine too.

Reply:
I don't know how to achieve that, so just keep it that way for now